README.md

```diff
@@ -1,14 +1,13 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2021S2-CS2103T-W12-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S2-CS2103T-W12-2/tp/actions)
 
 ![Ui](docs/images/Ui.png)
-
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
+  
+* This is **project for Software Engineering (SE) students**.<br>
+* The project simulates an ongoing software project for a desktop application (called _Car@Tinder_) <br>
+  used for managing car customers contacts .
+  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** <br>
+    (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* It is named `Car@Tinder` because we want car salesperson users to match their cars with the right customers.
```

Comment:
Hey, I actually changed the names from Car@Tinder to Car@leads as you told yesterday, can you reflect these changes?

Reply:
hey, I just changed to Car@leads psps